src/Microsoft.Health.Fhir.Core/Features/Persistence/TransactionHandler.cs

```diff
@@ -0,0 +1,36 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using EnsureThat;
+
+namespace Microsoft.Health.Fhir.Core.Features.Persistence
+{
+    public class TransactionHandler : ITransactionHandler
```

Comment:
Would the code still work if we required `IFhirDataStore` to implement `ITransactionHandler` and didn't have this class at all?

Reply:
Implemented as suggested